ansible/roles/apache-flink/templates/flink-init.j2

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# The path where Apache Flink is installed.
+INSTALLATION_PATH="{{ installation_path }}"
+
+# The full path of the pid file to use.
+PIDFILE="$INSTALLATION_PATH/flink/flink.pid"
+
+# The full path of the lock file to use.
+LOCKFILE="$INSTALLATION_PATH/flink/flink-lock"
+
+# The command that will start Apache Flink.
+START_COMMAND="$INSTALLATION_PATH/flink/bin/yarn-session.sh -n {{ number_of_taskmanagers }} -tm {{ ram_per_task_manager }}"
+
+# The command that will stop Apache Flink. Note that, HADOOP_HOME home variable should be set outside from this script and
+# before Flink's deployment.
+STOP_COMMAND="$HADOOP_HOME/bin/yarn application --kill"
```

Comment:
Yes but my question was about the boot process, not about the user login. What happens when the VM is rebooted? Furthermore the init scripts should not rely on the environment variables of the the user running the scripts. This why I am asking you to either source the /etc/environment or declare the variables properly